### ISSUE TITLE
ORBAT faction reverting after .pbo build

### ISSUE THREAD

**Issue by coaxjack** [ALiVEOS/ALiVE.OS#445]
This is a new issue for me, as I have made several factions before that worked just fine. Recently when I have been working on a new faction and compile the .pbo to maintain my progress, when I load it back into Arma some or all of the units have reverted their equipment to their dependencies' loadout inside ORBAT.

I'm basing my new faction using vanilla NATO dependencies. When looking at the actual config file, it also seems to have overwritten the new stuff I had exported originally with the dependencies' equipment.

TL;DR When exporting a faction, it's like ORBAT isn't saving anything, or not importing it correctly when loaded up again.

Edit: I should mention that the units that I had made on the first go (before I compiled the pbo to save where I was) DO actually work and appear correctly in Eden. It's just that when I go back into ORBAT to keep working on the factions they revert back to their default Nato gear.

**Comment by tupolov** [comment]
Fixed in 16a8e045b34551100a3364f9e2e1d0b8deb2cd0e
